components/ManagerSubmitButton.tsx

```diff
@@ -0,0 +1,54 @@
+import urlsForRouter from "helpers/urlsForRouter";
+import { Button } from "lbh-frontend-react";
+import { NextRouter, useRouter } from "next/router";
+import PropTypes from "prop-types";
+import React from "react";
+import PageSlugs, { urlObjectForSlug } from "steps/PageSlugs";
+
+interface Props {
+  onSubmit(router: NextRouter): Promise<void>;
+  buttonText: string;
+}
+
+const ManagerSubmitButton: React.FunctionComponent<Props> = (props) => {
+  const { onSubmit, buttonText } = props;
+  const router = useRouter();
+  const status = buttonText === "Approve" ? "approved" : "declined";
+
+  const { href, as } = urlsForRouter(router, {
+    ...urlObjectForSlug(router, PageSlugs.Confirmed),
+    query: { status: status },
+  });
+
+  return (
+    <Button
+      preventDoubleClick
+      className={
+        buttonText === "Decline"
+          ? "lbh-button--warning govuk-button--warning"
+          : "submit-button"
+      }
```

Comment:
There's a package we have installed to simplify this `classnames` (if we don't, I'm surprized, and it's worth installing).

Also, the `"submit-button"` class was for styling that we don't have here (but maybe should?). If you're seeing styling, it's likely a side effect of using global styles on `makeSubmit`, and we should use a different class name.

```suggestion
      className={
        classNames({ "lbh-button--warning govuk-button--warning": status === "decline" })
      }
```